code/modules/mob/living/simple_animal/bot/ed209bot.dm

```diff
@@ -54,21 +54,19 @@
 		lasercolor = created_lasercolor
 	icon_state = "[lasercolor]ed209[on]"
 	set_weapon() //giving it the right projectile and firing sound.
-	spawn(3)
```

Comment:
_Why?_

Reply:
This actually runtimed because it could not access the variables anymore for some reason trough the reason why this used spawn in the first place i can only speculate this was leftover after converting it to Initialize

Reply:
the relevant variables for most of the things below this lines are set in the parent of the initialize proc so they are there but again for some reasons it still runtimed in there even trough there was no obvious reason to